.travis.yml

```diff
@@ -1,2 +1,2 @@
 language: cpp
-script: mkdir build && cd build && cmake .. && make
+script: mkdir build && cd build && cmake .. && make && ./durian_test
```

Comment:
I think `cmake`, `make`, and running `./durian_test` at least should be on separate script steps to make Travis logs easier to read and make it easier to find which step failed

Reply:
I also believe you are missing `git submodule update --init --recursive`

Reply:
I've added a commit with an updated config @ 1888f2f